tests/test_mobilenet.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+import os
+import unittest
+from distutils.version import StrictVersion
+
+import numpy as np
+import keras
+import onnx
+import keras2onnx
+from keras.preprocessing import image
+from keras.applications.resnet50 import preprocess_input
+from common_test_class import CommonTestCase
+
+
+working_path = os.path.abspath(os.path.dirname(__file__))
+tmp_path = os.path.join(working_path, 'temp')
+
+
+class TestKerasMobileNet(CommonTestCase):
```

Comment:
Can we name this file test_keras_applications? We can add other keras applications to this file if needed. Also TestKerasMobileNet -> TestKerasApplications.

Reply:
Can we resolve this?